pipelines/rj_smtr/tasks.py

```diff
@@ -1275,6 +1299,36 @@ def transform_raw_to_nested_structure(
             # Check empty dataframe
             if data.empty:
                 log("Empty dataframe, skipping transformation...")
+
+            elif "customFieldValues" in data:
```

Comment:
apagar elif

if "customFieldValues" not in data:
                    for col in data.columns[data.dtypes == "object"].to_list():
                        data[col] = data[col].str.strip()